### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by Minuu7**
- [ ] I made sure that there are no linter errors.
- [ ] I made sure that you used the correct GitHub flow.
- [ ] I made sure that you documented your work in a professional way.
- [ ] I followed our list of best practices for HTML & CSS.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+h1 {
+  color: #00ff;
+  font-size: 50px;
+  display: flex;
+  justify-content: center;
```
Comment (mahbubul14) [inline style.css]: - [ ] You have a linter error on this project. Kindly fix the error. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Minuu7,

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

- [ ]  Use descriptive commit messages - each commit should be small enough to describe it in one sentence. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] Pull Request should have a descriptive title telling you what changes are inside.

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 1 above)
Comment (Minuu7) [reply style.css]: Hello Assalamualaikum. can you please check it out again now? I've corrected some errors that you mentioned. Thanks.

**Review by gemmen29 — APPROVED** [review]
Hi @Minuu7, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**